www/blog/2023/06/19/karmashapes/index.md

```diff
@@ -0,0 +1,790 @@
+
+Importing Karmashapes also allows any downstream consumer of WOF, like [Pelias](https://pelias.io/) via [Geocode.Earth](https://geocode.earth/) to benefit from the full scope of data Karmashapes provides. Let’s take a look at the locality and neighbourhood coverage now available in WOF:
+
+![Locality coverage in Kerala](images/kerala.png "Locality coverage in Kerala")
+_(above) Locality (blue) and neighbourhood (orange) coverage along the northern Kerala coast_
+
+![Locality coverage in Dehradun](images/dehradun.png "Locality coverage in Dehradun")
+_(above) Locality (blue) and neighbourhood (orange) coverage northwest of Dehradun_
+
+![Locality coverage in Chandigarh](images/chandigarh.png "Locality coverage in Chandigarh")
+_(above) Locality (blue) and neighbourhood (orange) coverage in and around Chandigarh_
+
+Above, you can see three areas of India - the northern Kerala coast, areas of Dehradun, and the greater Chandigarh area - with orange representing the neighbourhood placetype and blue representing the locality placetype. These example areas illustrate the complexity, density, and coverage that Karmashapes has added to Who’s On First.
+
+Below are two charts - the first is a per-placetype feature count in WOF before and after the Karmashapes import, the second is a more detailed breakdown of neighbourhood and locality placetype counts per region of India.
+
+<table>
+  <tr>
+   <td><strong>Placetype</strong>
+   </td>
+   <td><strong>WOF feature count, pre-Karmashapes</strong>
+   </td>
+   <td><strong>WOF feature count, post-Karmashapes</strong>
+   </td>
+  </tr>
+  <tr>
+   <td><em>Country</em>
+   </td>
+   <td><strong><em>1 total;</em></strong>
+<p>
+<em>1 polygon geometry</em>
+   </td>
+   <td><strong><em>1 total; 1 polygon geometry</em></strong>
+   </td>
+  </tr>
+  <tr>
+   <td><em>Region</em>
+   </td>
+   <td><strong><em>37 total; 37 polygons</em></strong>
+   </td>
+   <td><strong><em>37 total; 37 polygons</em></strong>
+   </td>
+  </tr>
+  <tr>
+   <td><em>County</em>
+   </td>
+   <td><strong><em>640 total; 640 polygons</em></strong>
+   </td>
+   <td><strong><em>640 total; 640 polygon</em></strong>
+   </td>
+  </tr>
+  <tr>
+   <td><em>Locality</em>
+   </td>
+   <td><strong><em>497,880 total; 1,546 polygons, 496,334 points</em></strong>
+   </td>
+   <td><strong><em>915,468 total; 553,804 polygons, 361,664 point</em></strong>
+   </td>
+  </tr>
+  <tr>
+   <td><em>Neighbourhood</em>
+   </td>
+   <td><strong><em>9,679 total:  1,302 polygons, 8,377 points</em></strong>
+   </td>
+   <td><strong><em>190,397 total; 25,312 polygons, 165,067 points</em></strong>
+   </td>
+  </tr>
+</table>
+
+_(above) A chart, per placetype, with pre and post Karmashapes feature counts, split by geometry type_
+
+<table>
```

Comment:
Let's convert tables to Markdown so they can render pretty using the WOF blog's CSS

Reply:
Consider Locality then Town in the column order

Reply:
done, and moved column order around